test/interval_arithmetic.jl

```diff
@@ -47,12 +47,11 @@ end
                       testcase.atoms, testcase.positions)
     basis = PlaneWaveBasis(model; Ecut=10, kgrid=(1, 1, 1))
 
-    eigenvalues = [[-0.17268859, 0.26999098, 0.2699912, 0.2699914, 0.35897297, 0.3589743],
-                   [-0.08567941, 0.00889772, 0.2246137, 0.2246138, 0.31941655, 0.3870046]]
+    eigenvalues = [[-0.17268859, 0.26999098, 0.2699912, 0.2699914, 0.35897297, 0.3589743]]
     occupations, εF = DFTK.compute_occupation(basis, Vector{Interval{Float64}}.(eigenvalues);
                                               occupation_threshold=1e-7)
```

Comment:
I think it would be nice to keep that with multiple k-points if possible, don't you think?